apps/analysis_framework/fixtures/af_roles.json

```diff
@@ -0,0 +1,67 @@
+[
+{
```

Comment:
Please use standard format (show that diff will make sense in the future)
Sample: https://github.com/the-deep/server/blob/develop/apps/ary/fixtures/ary_template_data.json

Reply:
Fixed